src/Bicep.Core/CoreResources.Designer.cs

```diff
@@ -8,218 +8,191 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-namespace Bicep.Core
-{
+namespace Bicep.Core {
```

Comment:
>Core [](http://example.com/codeflow?start=16&length=4)

nit - Why all the reformats? Also { without preceding newlines is more of a TypeScript formatting, which we generally don't do on C# side of things.

Reply:
The parent comment somehow isn't visible on GH, but can you take a look at this?

Reply:
There's a mismatch between what the ResGen tool writes out and the formatting that we're using in VS.  ResGen uses { on end of line, our formatting puts { on separate line.  This change actually puts it back in line with the other designer files, so I'll leave it this way and try to keep it that way in future.  It's a pain.